### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by taquitos**
Updated to >= API-31
cordova-android@11
Gradle 7.3.3

Fixes [CSDK-382]

[CSDK-382]: https://revenuecats.atlassian.net/browse/CSDK-382?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -224,23 +204,23 @@ jobs:
 
 workflows:
   version: 2
-  build-test:
-    jobs:
-      - runtest
+  # build-test:
+  #   jobs:
+  #     - runtest
```
Comment (vegaro) [inline .circleci/config.yml:209]: Don't forget to uncomment this

File: .circleci/config.yml [hunk 2]
```diff
@@ -224,23 +204,23 @@ jobs:
...
   ios-integration-test:
     jobs:
       - ios-integration-test: *release-tags-and-branches
 
-  api-tester-check:
-    jobs:
-      - api-tester
+  # api-tester-check:
+  #   jobs:
+  #     - api-tester
```
Comment (vegaro) [inline .circleci/config.yml:225]: And this

File: .circleci/config.yml [hunk 3]
```diff
@@ -224,23 +204,23 @@ jobs:
...
+  #     - runtest
   
   deploy:
     jobs:
       - docs-deploy: *release-tags
 
   android-integration-test:
     jobs:
-      - android-integration-test: *release-tags-and-branches
+      - android-integration-test
```
Comment (vegaro) [inline .circleci/config.yml:217]: And revert this

File: examples/cordova-sample/MyApp/wrapper.gradle [hunk 4]
```diff
@@ -0,0 +1 @@
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-bin.zip
```
Comment (vegaro) [inline examples/cordova-sample/MyApp/wrapper.gradle:1]: Does this work?

File: .circleci/config.yml (hunk 1 above)
Comment (taquitos) [reply .circleci/config.yml:209]: 🤦‍♂️

File: examples/cordova-sample/MyApp/wrapper.gradle (hunk 4 above)
Comment (taquitos) [reply examples/cordova-sample/MyApp/wrapper.gradle:1]: This was a bug in my script, deleting!

File: .circleci/config.yml [hunk 5]
```diff
@@ -135,14 +112,17 @@ jobs:
...
       - android/accept-licenses
       - android/create-avd:
           avd-name: myavd
-          system-image: system-images;android-29;default;x86
+          system-image: system-images;android-31;google_apis;x86_64
           install: true
       - android/start-emulator:
           avd-name: myavd
           no-window: true
+          wait-for-emulator: false
```
Comment (vegaro) [inline .circleci/config.yml:125]: Maybe we want this?

**Review by vegaro — APPROVED** [review]

File: .circleci/config.yml (hunk 5 above)
Comment (taquitos) [reply .circleci/config.yml:125]: It never completes 😿 
It just waits and waits and waits. It seems to be waiting for the logs to calm down and they never do. You can seen an example of this https://app.circleci.com/pipelines/github/RevenueCat/cordova-plugin-purchases/718/workflows/ae145a1b-0168-4293-b348-6dfb09d12621/jobs/1193

**Pull request merged**